package.json

```diff
@@ -90,7 +91,6 @@
     "jest": "^28.1.3",
     "playwright": "^1.24.0",
     "prettier": "^2.3.2",
-    "rimraf": "^3.0.2",
```

Comment:
Removing this here will potentially hose anyone doing local nativefier development outside of docker as some of our scripts (such as `clean`) require this devDependency.